### PR TITLE
New extensions list

### PR DESCRIPTION
**Pull request by kenzhemir**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,15 +19,16 @@
         "Extension Packs"
     ],
     "extensionDependencies": [
-        "EditorConfig.EditorConfig",
-        "ms-azuretools.vscode-docker",
```
Comment (kenzhemir) [inline package.json:23]: no need anymore

File: package.json [hunk 2]
```diff
@@ -19,15 +19,16 @@
...
-        "codezombiech.gitignore",
```
Comment (kenzhemir) [inline package.json:24]: no need anymore

File: package.json [hunk 3]
```diff
@@ -19,15 +19,16 @@
...
-        "waderyan.gitblame",
```
Comment (kenzhemir) [inline package.json:25]: gitlens comes with blame

File: package.json [hunk 4]
```diff
@@ -19,15 +19,16 @@
...
-        "esbenp.prettier-vscode",
-        "robertohuertasm.vscode-icons",
```
Comment (kenzhemir) [inline package.json:27]: too specific, also defaults are fine

File: package.json [hunk 5]
```diff
@@ -19,15 +19,16 @@
...
-        "sleistner.vscode-fileutils",
```
Comment (kenzhemir) [inline package.json:28]: not much use

File: package.json [hunk 6]
```diff
@@ -19,15 +19,16 @@
...
-        "waderyan.nodejs-extension-pack",
```
Comment (kenzhemir) [inline package.json:29]: added some of these separately, except snippets

File: package.json [hunk 7]
```diff
@@ -19,15 +19,16 @@
...
-        "snyk-security.snyk-vulnerability-scanner",
```
Comment (kenzhemir) [inline package.json:30]: eats resources, need to be opt-in

File: package.json [hunk 8]
```diff
@@ -19,15 +19,16 @@
...
-        "ryanluker.vscode-coverage-gutters"
```
Comment (kenzhemir) [inline package.json:31]: I don't think people use it much

File: package.json [hunk 9]
```diff
@@ -19,15 +19,16 @@
...
+        "christian-kohler.path-intellisense", 
+        "DavidAnson.vscode-markdownlint", 
+        "dbaeumer.vscode-eslint", 
+        "eamodio.gitlens", 
+        "EditorConfig.EditorConfig", 
+        "esbenp.prettier-vscode", 
+        "firsttris.vscode-jest-runner", 
+        "foxundermoon.shell-format", 
+        "jasonnutter.search-node-modules", 
+        "yoavbls.pretty-ts-errors"
```
Comment (kenzhemir) [inline package.json:32]: explainer for typescript errors

File: package.json [hunk 10]
```diff
@@ -19,15 +19,16 @@
...
-        "ryanluker.vscode-coverage-gutters"
+        "christian-kohler.npm-intellisense", 
+        "christian-kohler.path-intellisense", 
+        "DavidAnson.vscode-markdownlint", 
+        "dbaeumer.vscode-eslint", 
+        "eamodio.gitlens", 
+        "EditorConfig.EditorConfig", 
+        "esbenp.prettier-vscode", 
+        "firsttris.vscode-jest-runner", 
+        "foxundermoon.shell-format",
```
Comment (kenzhemir) [inline package.json:30]: for zsh development and shell script in general

File: package.json [hunk 11]
```diff
@@ -19,15 +19,16 @@
...
-        "waderyan.gitblame",
-        "esbenp.prettier-vscode",
-        "robertohuertasm.vscode-icons",
-        "sleistner.vscode-fileutils",
-        "waderyan.nodejs-extension-pack",
-        "snyk-security.snyk-vulnerability-scanner",
-        "ryanluker.vscode-coverage-gutters"
+        "christian-kohler.npm-intellisense", 
+        "christian-kohler.path-intellisense", 
+        "DavidAnson.vscode-markdownlint",
```
Comment (kenzhemir) [inline package.json:24]: markdown linter for readme's

**Review by saarmae — APPROVED** [review]

**Pull request merged**